src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/entity/CalendarId.java

```diff
@@ -14,6 +14,26 @@
     @JoinColumn(name="merged_calendar_id")
     MergedCalendar mergedCalendar;
 
+    @Transient
+    SharingOptions sharingOptions;
```

Comment:
@matheb you can leave this out, if you have a `@JsonSetter` attribute added to the `setter` method of this